### ISSUE TITLE
Can’t remap controller hotkeys in Hotkeys settings

### ISSUE THREAD

**Issue by xoredg** [yuzu-emu/yuzu#11986]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Affected Build(s)

Mainline / EA 3969

### Description of Issue

In Emulation -> Configure -> Hotkeys I am not able to remap a function by pressing controller keys. I’ve already picked my controller in the Controls section, but this window seems unable to register any controller button presses, requiring manual and error-prone edits of the config file to rebind anything. The need for that is quite common with users of Xbox Controller interfaces. 

For context, Steam Input or any other controller layers are not present, and my controller seems to behave ok in Yuzu UI (e.g controller navigation works). 

### Expected Behavior

I should be able to double click a function, see the Waiting for rebind, press the buttons on my controller, and having them be registered as a controller binding. 

### Reproduction Steps

1. Optional, as I don’t know if needed but you need to do this for games anyway: pick correct controller in Emulation - Configure - Controls - Player 1 - Input Device
2. Go to Emulation - Configure - General - Hotkeys
3. Double click any action row on the right side where the Controller Hotkeys column is
4. Press the desired controller buttons after [waiting] appears
5. In my case, this just reverts back to default as if no buttons were pressed. 

### Log File

I can’t capture logs at the moment, but it should be fairly easy to repro. 

### System Configuration

CPU: Ryzen 5950X and Z1 Extreme on ROG Ally
GPU/Driver: Nvidia latest / ASUS AMD latest 
RAM: 16GB/32 GB
OS: Windows 11 Pro

**Comment by danielcherubini** [comment]
Are you also noticing that your assigned controller hotkeys don't work also? 

I can't get any mapped hotkeys to function. They do in game, but it seems Yuzu isn't recognising them. I have quit set to plus + minus, and it doesn't work..

**Comment by xoredg** [comment]
Yes, the default hotkeys don't get recognized, but I assumed it has due to my device controller on that case. Editing the config manually (which is very cumbersome), while yuzu is closed and assigning the desired hotkeys does seem to work once restarted.

---
Best Regards,

Adrian Perez
https://adrianperez.codes
________________________________

**Comment by mattebad** [comment]
Would like to also confirm I'm running into the same issue on Yuzu 1664. Like was said above, both remapping the hotkeys and the actual execution of the hotkey does not work. Its like the input isn't picked up by Yuzu at all even though the input is picked up in game. I do have steam open in the background but even with that closed it doesn't change anything.

System Configuration
CPU: Ryzen 5900x
GPU/Driver: RTX 3070/Nvidia latest
RAM: 32GB
OS: Windows 10 Pro

**Comment by zhen-huan-hu** [comment]
Same issue here. Controller is recognized by yuzu but mapping controller hotkeys does not work in the interface. Had to edit the config file directly.

System Configuration 1:
CPU: i5 3570
CPU/Driver: AMD 6700XT/latest
RAM: 16GB
OS: Windows 10 Pro

System Configuration 2:
Steam Deck OLED 512GB

**Comment by dzemp** [comment]
Same issue with Lenovo Legion Go.
